packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx

```diff
@@ -95,6 +122,11 @@ export const MetricsTable = () => {
         return lastPage.pagination?.totalResults ?? 0;
     }, [data]);
 
+    const showLoadingOverlay = useMemo(
+        () => isFetching && isPreviousData && !isMutating,
```

Comment:
Can't we just use `isFetchingNextPage` here?

Reply:
apparently not lol...